Sources/StreamChatUI/ChatMessageList/ChatThreadVC.swift

```diff
@@ -206,39 +188,44 @@ open class _ChatThreadVC<ExtraData: ExtraDataTypes>:
         layoutOptions.remove(.threadInfo)
         return layoutOptions
     }
+    
+    public var messages: [_ChatMessage<ExtraData>] {
+        messageController.replies + [messageController.message!]
+    }
+    
+    open func numberOfSections(in tableView: UITableView) -> Int {
+        1
+    }
```

Comment:
Not needed

Reply:
https://github.com/GetStream/stream-chat-swift/pull/1219#discussion_r659803576